src/components/Blog/Blog.css

```diff
@@ -154,6 +154,13 @@ pre{
     word-wrap: break-word;
     white-space: pre;
     font-size: 16px;
+	display :block;
+}
+table{
```

Comment:
@PrP-11 Please use classes or specific attributes. Make sure you are not changing anything else in the application.